packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,322 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { CreateStyled } from '@emotion/styled';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { Step, StepProps, StepStatus } from './Step';
+import { transparentize } from 'polished';
+import { I18nContext } from '../../i18n';
+
+export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * @internal
+   */
+  testId?: string;
+  /**
+   * Custom number for responsive styling, uses a minimum 'breakpoint' width from the set number.
+   */
+  breakpoint?: number;
+  /**
+   * Changes the Stepper view for responsive layouts, needs a minimum 'breakpoint' number.
+   */
+  breakpointLayout?: StepperLayout;
+  /**
+   * Sets the Stepper view
+   * @default StepperLayout.showLabels
+   */
+  layout?: StepperLayout;
+  /**
+   * Sets a custom label for the Step count # of #.
+   * @default 'Step'
+   */
+  stepLabel?: string;
+  /**
+   * Sets a custom label for the completed Step.
+   * @default 'All steps completed'
+   */
+  completionLabel?: string;
+  /**
+   * Current step value.
+   */
+  currentStep?: number;
```

Comment:
We should probably make `currentStep` required. This whole thing doesn't do anything without it